### ISSUE TITLE
tmux-yank crashes tmux on my Fedora system

### ISSUE THREAD

**Issue by Notgnoshi** [Notgnoshi/dotfiles#116]
* wayland
* wl-copy and xclip are both installed
* plugin is up-to-date

Selecting anything immediately crashes

**Comment by Notgnoshi** [comment]
This appears to have nothing to do with tmux-yank. It happens with JUST the following in my `~/.tmux.conf`

```
set -g mouse on
```

**Comment by Notgnoshi** [comment]
Resolved by building and installing tmux 3.4 from source

```sh
sudo dnf install ncurses-devel libevent-devel
git clone https://github.com/tmux/tmux.git
cd tmux
git checkout 3.4
sh autogen.sh
./configure --prefix $HOME/.local/
make
make install
```
